### ISSUE TITLE
I can't open directory in repository

### ISSUE THREAD

**Issue by kodermax** [Alfresco/alfresco-ng2-components#1572]
**Type of issue:**  (check with "[x]")
```
- [ ] New feature request
- [X] Bug  
- [ ] Support request
```
**Current behavior:**
When I open directory in repository I see error.
![image](https://cloud.githubusercontent.com/assets/590985/22421974/1a26d6d2-e6fb-11e6-872f-f9a8bb141d7b.png)
![image](https://cloud.githubusercontent.com/assets/590985/22421980/208effb8-e6fb-11e6-9d09-6549d3e18868.png)
![image](https://cloud.githubusercontent.com/assets/590985/22421989/26f5dffc-e6fb-11e6-863c-e8e3c3d6cf46.png)

**Expected behavior:**

**Steps to reproduce the issue:**
When I open root paths
![image](https://cloud.githubusercontent.com/assets/590985/22422071/9571e5a2-e6fb-11e6-8bbc-7faa7191717f.png)

I get error.

**Component name and version:** 
"ng2-alfresco-documentlist": "1.0.0",

**Browser and version:** 
Chrome Latest version
 
**Node version (for build issues):**  
7.2.0

**Comment by DenisVuyka** [comment]
Notes: relative paths are no longer used starting with 1.1.0, so this issue may not be reproduced with latest codebase.

**Comment by DenisVuyka** [comment]
The issue was raised for 1.0.0, does it still occur for 1.2.0 one?

**Comment by jdosti** [comment]
Solved with the 1.2.0 release.
